src/clarity-angular/emphasis/alert/index.ts

```diff
@@ -1,12 +1,15 @@
 /*
- * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
+
 import { Type } from "@angular/core";
 import { Alert } from "./alert";
+import { AlertItem } from "./alert-item";
 
 export * from "./alert";
```

Comment:
The standard is to also `export * from "./alert-item"` here. Otherwise consumers have no way to get the actual `AlertItem` class if they need to.